packages/kubernetes/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.41.0"
+  changes:
+    - description: Enabling GA release for time_series index.mode
```

Let's have a more user friendly description here:
```suggestion
    - description: Enable time series data streams for the metrics datasets. This improves storage usage and query performance. For more details, see https://www.elastic.co/guide/en/elasticsearch/reference/master/tsds.html
```

---

Updated!